app/src/main/kotlin/nyc/jsjrobotics/palmrgb/firebase/constants/FirebaseDatabaseReference.kt

```diff
@@ -0,0 +1,11 @@
+package nyc.jsjrobotics.palmrgb.firebase.constants
+
+import com.google.firebase.database.FirebaseDatabase
+
+
+object FirebaseDatabaseReference {
```

Comment:
What is the point of this class? It seems like it does nothing that another class couldnt do by calling the same static functions